src/components/tl/ucp/allgather/allgather.h

```diff
@@ -74,6 +75,9 @@ ucc_status_t ucc_tl_ucp_allgather_sparbit_init(ucc_base_coll_args_t *coll_args,
                                                 ucc_base_team_t      *team,
                                                 ucc_coll_task_t     **task_h);
 
+/* XGVMI */
+void ucc_tl_ucp_dpu_xgvmi_rdma_progress_allgather(ucc_coll_task_t *coll_task);
```

Comment:
```suggestion
void ucc_tl_ucp_dpu_allgather_xgvmi_rdma_progress(ucc_coll_task_t *coll_task);
```
to keep a pattern "tl_coll_algo_(init|start|progress|finalize)"